### ISSUE TITLE
Where is the "BatchNormReimpl"?

### ISSUE THREAD

**Issue by skanderbug** [vacancy/Synchronized-BatchNorm-PyTorch#29]
sync_batchnorm/batchnorm_reimpl.py
__all__ = ['BatchNormReimpl']

There is no BatchNormReimpl in batchnorm_reimpl.py.
Maybe BatchNorm2dReimpl instead?

**Comment by vacancy** [comment]
Fixed by 0d261dd. Thanks for reporting.
